packages/language-server/src/plugins/typescript/module-loader.ts

```diff
@@ -49,6 +50,7 @@ class ModuleResolutionCache {
         this.cache.forEach((val, key) => {
             if (val && this.getCanonicalFileName(val.resolvedFileName) === resolvedModuleName) {
                 this.cache.delete(key);
+                this.pendingInvalidations.add(key.split(':::').shift() || '');
```

Comment:
Probably makes sense to have `:::` as `const` at the top and use that at this point